.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+Contributing
+=
+
+All the contributions are very welcome!
+
+We are following the [GitHub Flow Guide](https://docs.github.com/en/get-started/quickstart/github-flow). As a resume to the steps to be taken, here is a list of them:
+
+1. Fork the main repository and clone to your local working area
```

Comment:
Indicate to fetch upstream if you forked before: Fork the main repository and clone or fetch upstream and pull (if you forked before) to your local working area